config/types.go

```diff
@@ -141,7 +141,7 @@ type SecretStoreInfo struct {
 	Type           string
 	Host           string
 	Port           int
-	Path           string
+	SecretName     string
```

Comment:
This should be StoreName since it is the name given to the Service secret store, which is always the service's service key.